### PR TITLE
DiseasesExpanded - FragrantFlowers germs support added to Germcatcher

### PR DESCRIPTION
**Pull request by realwiseserpent**
Hi! Now Germcatcher creates pollen flask from Fragrant Flowers germs.

### CODE REVIEW

File: DiseasesExpanded/Buildings/Germcatcher.cs [hunk 1]
```diff
@@ -45,6 +45,10 @@ public static void InitalizeFlaskDict()
                 SpawnedFlasks.Add(GermIdx.BogInsectsIdx, BogBugsFlask.ID);
             if (DlcManager.IsExpansion1Active() && Settings.Instance.HungerGerms.IncludeDisease)
                 SpawnedFlasks.Add(GermIdx.HungerGermsIdx, HungermsFlask.ID);
+
+            SpawnedFlasks.Add(GermIdx.RoseScentIdx, PollenFlask.ID);
+            SpawnedFlasks.Add(GermIdx.LavenderScentIdx, PollenFlask.ID);
+            SpawnedFlasks.Add(GermIdx.MallowScentIdx, PollenFlask.ID);
```
Comment (pether-pg) [inline DiseasesExpanded/Buildings/Germcatcher.cs:51]: Add bool variable to ModInfo that informs if FragrantFlowers is enabled. Use CheckForRelatedMods to initalize the variable. surround this block of code with if(ModInfo.IsFragrantFlowersEnabled)

File: DiseasesExpanded/Misc/GermIdx.cs [hunk 2]
```diff
@@ -138,14 +138,45 @@ public static byte MedicalNanobotsIdx
                 return _medNanobots;
             }
         }
+        //---------- FragrantFlowers ----------
```
Comment (pether-pg) [inline DiseasesExpanded/Misc/GermIdx.cs:141]: add extra line above

**Review by pether-pg — CHANGES_REQUESTED** [review]

File: DiseasesExpanded/Buildings/Germcatcher.cs (hunk 1 above)
Comment (realwiseserpent) [reply DiseasesExpanded/Buildings/Germcatcher.cs]: Fixed.

File: DiseasesExpanded/Misc/GermIdx.cs (hunk 2 above)
Comment (realwiseserpent) [reply DiseasesExpanded/Misc/GermIdx.cs:142]: Fixed.

**Review by pether-pg — APPROVED** [review]

**Pull request merged**